button-card.js

```diff
@@ -303,58 +306,41 @@
       return 3;
     }
 
-    _toggle(state, config) {
-      if (!config.tap_action)
-        config.tap_action = 'toggle'
+    _handleTap(state, config) {
       if (config.tap_action) {
+        let event;
         switch (config.tap_action.action) {
           case 'none':
             break;
           case 'more-info':
-          case 'more_info': {
-            const node = this.shadowRoot;
-            const options = {};
-            const detail = { entityId: state.entity_id };
-            const event = new Event('hass-more-info', {
-              bubbles: options.bubbles === undefined ? true : options.bubbles,
-              cancelable: Boolean(options.cancelable),
-              composed: options.composed === undefined ? true : options.composed,
+          case 'more_info':
+            event = new Event('hass-more-info', {
```

Comment:
`event` is not needed outside the individual cases. Leave it as a `const` for each case that needs it

Reply:
I changed it like this because without the braces on the `case` block, the scope of variables are at the `switch` level and not at each `case` level so the browser was complaining about `event` being already defined...